### ISSUE TITLE
SOAPAction Header not being generated

### ISSUE THREAD

**Issue by banderson5144** [pepri/wizdler#8]
Hi,

I do a lot of work with the Salesforce API and I really liked your tool. However, its not generating the SOAPAction header correctly. If you want i can send the WSDL.

**Comment by pepri** [comment]
Hello,

send me please the WSDL, so I can look into it. Meanwhile you can use the the HTTP headers editor, which was added in version 1.11. The HTTP headers editor is available after clicking the "HTTP Headers" in the menu near the Go button.

**Comment by banderson5144** [comment]
Hey,

I actually figured out what the problem is. Salesforce expects a SOAPAction parameter with an empty string (SOAPAction:""). So locally I made this change to the wsdl.js file at line 130:

request.headers['SOAPAction'] = ctx.operation.soap.action == '' ? '""' : ctx.operation.soap.action;

Also thanks for adding the HTTP Header thing. However, I have updated the extension and I still don't see the HTTP Headers option in the menu.
